lib/nifty_services/support/hash.rb

```diff
@@ -0,0 +1,10 @@
+unless Hash.method_defined?(:symbolize_keys)
+  class Hash
+   def symbolize_keys
```

Comment:
This is a destructive! method, right? We just need to return a copy of `self` with changes, see this: 

```ruby
hash = { 'a' => 10, 'b' => 20, 'c' => 30 }
hash.symbolize_keys # {:a=>10, :b=>20, :c=>30}
# now original reference its destructed, this isnt good in anyway!
hash # {:a=>10, :b=>20, :c=>30}
```

This can be dangerous! I'm not safe merging this